### PR TITLE
features:*

### PR DESCRIPTION
**Pull request by dickeyxxx**

### CODE REVIEW

**Comment by dickeyxxx** [comment]
ready for review @ransombriggs

File: commands/features/disable.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+let cli = require('heroku-cli-util');
+let co  = require('co');
+
+function* run (context, heroku) {
+  let app     = context.app;
+  let feature = context.args.feature;
+
+  yield cli.action(`Disabling ${cli.color.green(feature)} for ${cli.color.cyan(app)}`, co(function* () {
```
Comment (ransombriggs) [inline commands/features/disable.js]: @dickeyxxx shouldn't this work for account features as well?

Comment (ransombriggs) [inline commands/features/disable.js]: @dickeyxxx never mind, I am looking at the ruby code for features listing and it makes more sense now.

**Comment by ransombriggs** [comment]
@dickeyxxx LGTM

**Pull request merged**